Task 3/binary_search_recursive_approach/question.md

```diff
@@ -0,0 +1,47 @@
+# [Q] Serach element using binary search using recursion
```

Comment:
Task 3 does not require a markdown file. You can add the input output as a part of a multiline comment in the beginning, or move this program to Task 1.

Reply:
okay